src/factories/versioning-strategy-factory.ts

```diff
@@ -40,6 +41,7 @@ const versioningTypes: Record<string, VersioningStrategyBuilder> = {
   'always-bump-minor': options => new AlwaysBumpMinor(options),
   'always-bump-major': options => new AlwaysBumpMajor(options),
   'service-pack': options => new ServicePackVersioningStrategy(options),
+  prerelease: options => new PrereleaseVersioningStrategy(options),
```

Comment:
Should `prerelease` be in quotes?

Reply:
`Record` types in like with string keys are set with or without quotes in typescript. The linter/formatter will enforce it to be quoteless if quotes are not necessary.